src/pages/index.js

```diff
@@ -157,14 +161,9 @@ function handleDeleteSubmit(evt) {
 
   api
     .deleteCard(selectedCardId)
-    .then(() => {
-      selectedCard.remove();
-      closeModal(deleteModal);
-    })
+    .then(selectedCard.remove(), closeModal(deleteModal))
```

Comment:
Please, return the right code back. You shouldn't have changed the other places with `() =>`. They are needed.